doc/command_ref.rst

```diff
@@ -1068,7 +1065,7 @@ This command by default does not force a sync of expired metadata. See also :ref
 Repoquery Command
 -----------------
 
-``dnf [options] repoquery [<select-options>] [<query-options>] [<pkg-spec>]``
+``dnf [options] repoquery [<select-options>] [<query-options>] [<package-name-spec>]``
```

Comment:
I believe that this is incorrect. Repoquery accepts more then package-name-spec arguments like `/etc/dnf/dnf.conf`